.gitignore

```diff
@@ -16,3 +16,4 @@
 coverage.txt
 codegpt
 dist
+.idea/*
```

Comment:
Unfortunately, the code patch provided is not valid code and cannot be reviewed. Can you please provide a valid code snippet?